### PR TITLE
Exams by Kroshka Illia IM-23

### PR DESCRIPTION
**Pull request by illiakroshka**

### CODE REVIEW

**Comment by v-jester** [comment]
Good job, just too many commits

File: Tasks/day.js [hunk 1]
```diff
@@ -1,15 +1,11 @@
-// Get day number
+'use strict';
 
-let D = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+const Days = ['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat'];
```
Comment (v-jester) [inline Tasks/day.js:3]:
```suggestion
const DAYS = ['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat'];
```
The Days is a global constant and must be written in caps.

File: Tasks/merge.js [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
-  for (attribute_name in object_2) {
-    object_3[attribute_name] == object_2[attribute_name];
-    object_3[attribute_name] = object_2[attribute_name];
-  }
-  return object_1, object_2, object_3;
+const mergeTwoObjects = (object1, object2) => {
+  const object3 = {};
+  const addTo = (object) => {
+    for (const [key, value] of Object.entries(object))
+      object3[key] = value;
```
Comment (tshemsedinov) [inline Tasks/merge.js:7]:
```suggestion
    for (const [key, value] of Object.entries(object)) {
      object3[key] = value;
    }
```

File: Tasks/merge.js [hunk 3]
```diff
@@ -1,16 +1,15 @@
...
-    object_3[attribute_name] == object_1[attribute_name];
-    object_3[attribute_name] = object_1[attribute_name];
-  }
-  for (attribute_name in object_2) {
-    object_3[attribute_name] == object_2[attribute_name];
-    object_3[attribute_name] = object_2[attribute_name];
-  }
-  return object_1, object_2, object_3;
+const mergeTwoObjects = (object1, object2) => {
+  const object3 = {};
```
Comment (tshemsedinov) [inline Tasks/merge.js:4]: Naming hint: `result`

File: Tasks/merge.js [hunk 4]
```diff
@@ -1,16 +1,15 @@
...
+  const addTo = (object) => {
```
Comment (tshemsedinov) [inline Tasks/merge.js:5]: Unneeded complexity, can be easily solved without extra closure

File: Tasks/merge.js [hunk 5]
```diff
@@ -1,16 +1,15 @@
...
+      object3[key] = value;
+    return object3;
+  };
+  addTo(object1);
+  addTo(object2);
+  return object3;
 };
 
-require('../Tests/merge.js')(merge_two_objects);
+require('../Tests/merge.js')(mergeTwoObjects);
```
Comment (tshemsedinov) [inline Tasks/merge.js:15]: Add empty line at EOF

**Review by tshemsedinov — APPROVED** [review]

**Pull request closed without merging**